Example/UserAccountsView.swift

```diff
@@ -44,86 +27,45 @@ extension Font {
 /**
 	Main view of User Accounts preference screen
  */
-struct UserAccountsView: View {
+struct UserAccountsView: View, PreferencePaneView {
 	@State private var isOn1 = true
 	@State private var isOn2 = false
 	@State private var selection1 = 1
 	@State private var selection2 = 0
-	let width: CGFloat = 510.0
+	let contentWidth: CGFloat = 450.0
+	let preferencePaneIdentifier: PreferencePaneIdentifier = .userAccounts
+	let preferencePaneTitle: String = "User Accounts"
+	let toolbarItemIcon: NSImage = NSImage(named: NSImage.userAccountsName)!
         
 	var body: some View {
-		VStack(alignment: .preferenceSectionLabel) {
-			CheckboxSectionView(isOn1: $isOn1, isOn2: $isOn2)
-			Divider()
-				.frame(width: width, height: 20.0)
-				.alignmentGuide(.preferenceSectionLabel, computeValue: { $0[.leading] + 104.0 })
-			RadioSectionView(selection: $selection1)
-			Divider()
-				.frame(width: width, height: 20.0)
-				.alignmentGuide(.preferenceSectionLabel, computeValue: { $0[.leading] + 104.0 })
-			PickerSectionView(selection: $selection2)
-		}
-		.frame(width: width, alignment: .leading)
-		.padding([.top, .bottom], 20.0)
-		.padding([.leading, .trailing], 30.0)
-	}
-}
-
-// Some example sections with various input methods
-struct RadioSectionView: View {
-	@Binding var selection: Int
-    
-	var body: some View {
-		HStack(alignment: .top) {
-			Text("Show scroll bars:")
-				.alignmentGuide(.preferenceSectionLabel, computeValue: { $0[.trailing] })
-			Picker(selection: $selection, label: EmptyView()) {
-				Text("When scrolling").tag(0)
-				Text("Always").tag(1)
+		Preferences(contentWidth: contentWidth) {
+			PreferenceSection(title: "Permissions:") {
+				VStack(alignment: .leading) {
+					Toggle("Allow user to administer this computer", isOn: self.$isOn1)
+					Text("Administrator has root access to this machine.")
+						.font(.preferenceDescription)
+						.foregroundColor(.secondary)
+					Toggle("Allow user to access every file", isOn: self.$isOn2)
+				}
 			}
-				.pickerStyle(RadioGroupPickerStyle())
-			Spacer()
-		}
-	}
-}
-
-struct CheckboxSectionView: View {
-	@Binding var isOn1: Bool
-	@Binding var isOn2: Bool
-    
-	var body: some View {
-		HStack(alignment: .top) {
-			Text("Permissions:")
-				.alignmentGuide(.preferenceSectionLabel, computeValue: { $0[.trailing] })
-			VStack(alignment: .leading) {
-				Toggle("Allow user to administer this computer", isOn: $isOn1)
-				Text("Administrator has root access to this machine.")
-					.font(.preferenceDescription)
-					.foregroundColor(.secondary)
-				Toggle("Allow user to access every file", isOn: $isOn2)
+			PreferenceSection(title: "Show scroll bars:") {
```

Comment:
Like with most other views, `PreferencesSection` should accept both a title and a view. This is necessary when you need something else than just `Text()` for the title. Example:

<img width="780" alt="Screenshot 2020-02-11 at 23 12 53" src="https://user-images.githubusercontent.com/170270/74255148-0a963900-4d24-11ea-87af-5db64d3374e4.png">